docs/UserGuide.md

```diff
@@ -282,40 +284,51 @@ Examples: <br>
 
-* Multiple fields can be provided.<br>
+* Multiple search fields may be provided.<br>
 
-* Query is case-insensitive but space-sensitive.<br>
-</div>
+* Queries are case-insensitive but space-sensitive.<br>
 
 * For search queries regarding `salary`, `price` and `commission`:
-  * if you would like to search for exact `salary`/`price`/`commission`, you can follow this format (e.g. `$[Number]/hr`, `$[Number]/bag`). Only results that match the exact number will be shown. Examples:
+  * if you would like to search for the exact `salary`/`price`/`commission`, you may follow this format:<br>
+    * For `salary` and `commission`: `$[Number]/hr`<br>
+    * For `price`: `$[Number]/[quantifier]`<br>Only results that match the exact `Number` will be shown.<br>
+  * If you would like to search for `salaries`/`prices`/`commissions` within a specified range instead, you may follow this format:<br>
+    * For `salary` and `commission`: `$[Query digits]/hr`<br>
+    * For `price`: `$[Query digits]/[quantifier]`<br>In this case, `Query digits` refer to the first few digits of a number.
+      * For instance, if you would like to search for all salaries in the range `$50-59/hr`, you may key in `/search ; salary : $5`, with `$5` as the search query. This displays all staff that are paid from `$50/hr` to `$59/hr`.
+      * Note that this query also searches for staff with salaries of exactly `$5/hr`, from `$500/hr` to `$599/hr` as well, and even in larger ranges too.<br>
+    * This same query type also applies to `price` and `commission` fields.<br>
+    * `quantifier` refers to counter terms by which products are sold in.<br>
+    * Examples of valid `quantifier` values:
+      * "bag", "sack", "gram", "litre" etc.
+    * `quantifier` may assume any non-empty string.
+  * Examples of valid search commands:
     * `/search ; salary : $50/hr`
-    * `/search ; price : $50/bag`
-  * If you want to search for `salaries`/`prices`/`commissions` within a specified range instead, you can follow this format `$[Query digits]`. In this case, query digits refer to the first few digits of a number.
-    * For instance, if you want to query for all salaries in the range `$50-59/hr`, you can key in `/search ; salary : $5`, with `$5` as the search query. This displays all staff that are paid from `$50/hr` to `$59/hr`.
-    * Note that this query also returns staff with salaries exactly `$5/hr`, from `$500/hr` to `$599/hr` as well, and even in larger ranges too. This same query type also applies to `price` and `commission`.
-
-* Keying in repeated fields (e.g. )
+    * `/search ; price : $50/bag`<br>
+* Keying in repeated fields will result in an error message as follows:<br>
+  `Multiple values specified for the following single-valued field(s): ; [duplicated field] :`<br>
+  * Example of a search command with a duplicated search field:
+    * `/search ; name : Pooch ; name : Pooch`
+</div>
 
```

Comment:
deos this search need the duplicate field line too? the for your convenience ...

Reply:
resolvedd